lib/specifications/Specification.js

```diff
@@ -1,11 +1,21 @@
-const resourceFactory = require("@ui5/fs").resourceFactory;
-
+import logger from "@ui5/logger";
+import * as resourceFactory from "@ui5/fs/resourceFactory";
```

Comment:
only one method of the resourceFactory is used. Therefore use the named import not all.

Reply:
Done